src/main/java/com/nha/abdm/wrapper/hrp/mongo/tables/RequestLogs.java

```diff
@@ -28,12 +28,14 @@ public class RequestLogs {
 
     @Field("abhaAddress")
     public String abhaAddress;
```

Comment:
I am assuming the `patientReference` is the hospital Reference Id for patient and shouldn't we use this to link in all the tables. This field in the `Patient` table can be made unique as well.